modules/nf-core/gtfsort/main.nf

```diff
@@ -0,0 +1,49 @@
+process GTFSORT {
+    tag "$gtf"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gtfsort:0.2.2--h4ac6f70_0':
+        'biocontainers/gtfsort:0.2.2--h4ac6f70_0' }"
+
+    input:
+
+    path gtf
```

Comment:
I would suggest adding a meta map (with meta of [id]) to make it easier to use with multiple inputs and outputs.

Reply:
I was also perplexed about the same thing. However, from my understanding, gtf files are usually used as a reference file (see pipelines rnaseq,scrnaseq) and right now I'm not sure a can think  of a pipeline that requires more than one annotation file. Do you have anything specific in mind? If the norm is to use only one file, wouldn't make things more complicated if I added a meta map?

Reply:
If you're confident that people will not use it with more than one input, feel free to leave it as is.